### PR TITLE
Add remaining ship modules: Auto-Nav, Cargo Mgmt, Entertainment, Wellness Suite

### LINKED ISSUE #26: Remaining modules: Auto-Nav, Cargo Mgmt, Entertainment, Medical Bay
## Overview

Add the four non-drone modules. Depends on module infrastructure from #25.

### Auto-Nav Computer (800€)
- If nav check is due today (`TripDay > 0 and TripDay mod 3 == 0`), auto-complete it in `next_day()`
- Print notification. Nav check task doesn't appear in P3 that day
- Full effect at 75%+, reduced at 50-74% (maybe only auto-completes every other check?), offline below 50%

### Cargo Management (500€)
- Auto-file 1 paperwork chunk per day: `PaperworkDone = MIN(PaperworkDone + 1, PaperworkTotal)`
- Print notification if there was paperwork to file
- Full effect at 75%+, reduced at 50-74%, offline below 50%

### Entertainment System (400€)
- Full (75-100%): existing recreation tasks give +50% morale bonus. New options: "Play video games" (1 AP, +10 morale), "Listen to music" (1 AP, +5 morale)
- Reduced (50-74%): new options available but no bonus to existing tasks
- Below 50%: offline, no new options, no bonus
- AP costs should NOT decrease

### Medical Bay (900€)
- Wire `has_medical_module()` → `is_module_active(MedBay)` (replacing the current stub that returns false)
- Active (50%+): improves medical emergency outcomes (75% improve vs 50%, no death possible)
- Below 50%: returns false, no benefit

## Design reference
Full details in `~/.claude/plans/deep-drifting-leaf.md` (Module auto-complete behavior section)

## Files to modify
1. **`modules.ink`** — add module data entries (created in #25)
2. **`ship.ink`** — auto-complete logic in `next_day()` for Auto-Nav and Cargo Mgmt, entertainment options in relax sub-menu
3. **`events.ink`** — replace `has_medical_module()` stub
4. **`docs/`** — document new modules

### PR DESCRIPTION
**Pull request by spaceninja**
Closes #26

## Summary

Adds the four non-drone ship modules, completing the module system:

- **Auto-Nav Computer** (800€) — auto-completes navigation checks every 3 days. Full (75%+): every check. Reduced (50-74%): every other check. Suppresses the nav check P3 task when complete.
- **Cargo Management System** (500€) — auto-files one paperwork chunk per day. Full (75%+): every day. Reduced (50-74%): odd-numbered trip days only. Suppresses the paperwork P3 task when done.
- **Entertainment System** (400€) — all recreation gives +50% morale at full condition (75%+). Two new P4 tasks unlock when active (50%+): "Play video games" (+10 morale) and "Listen to music" (+5 morale). Centralized via `apply_recreation_bonus()` in ship.ink.
- **Wellness Suite** (500€) — combined gym, autodoc, drug printer, sunlight simulator, remote therapy, and hair trimmer. Daily passive: -5 fatigue / +2 morale at full, -3 / +1 at reduced. Also wires `has_medical_module()` → `is_module_active(WellnessSuite)`, improving medical emergency outcomes.

All four modules follow the existing graduated effectiveness pattern (full at 75%+, reduced at 50-74%, offline below 50%) and integrate with the existing auto-tasks system via a new `module_auto_tasks` tunnel.

## Test plan

- [x] `npm run lint` passes
- [x] `npm test` passes (417 tests, +37 new)
- New unit tests: `ModuleData` prices/names, getter/setter round-trips, `has_medical_module`, `apply_recreation_bonus` (including integer math edge cases)
- New integration tests: Auto-Nav at full/reduced/offline, nav task suppression, Cargo Mgmt at full/reduced/offline/complete, paperwork task suppression, Wellness Suite fatigue/morale at full/reduced/offline, Entertainment System task availability

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: docs/developer-guide.md [hunk 1]
```diff
@@ -441,6 +441,18 @@ Modules follow the same LIST + function lookup pattern as cargo, locations, and
 
 Drones run as a tunnel (`drone_auto_tasks`) called from `next_day()` (after settle → age → add daily tasks) and at trip start (after `generate_backlog()`). They prefer stale tasks over fresh ones.
 
+### Non-Drone Modules
+
+Non-drone modules run via a separate `module_auto_tasks` tunnel, called immediately after `drone_auto_tasks` in both `next_day()` and at trip start.
+
+- **Auto-Nav Computer** (800€) — auto-completes nav checks (due every 3 days). Full (75%+): every check. Reduced (50-74%): every other check (when `NavChecksCompleted mod 2 == 0`). The existing nav check task condition naturally suppresses the P3 task when already completed.
```
Comment (spaceninja) [inline docs/developer-guide.md]: Something I just considered. Instead of skipping every other check, what if we gave the user a bit of feedback like "The auto-nav is acting up again. It wasn't able to handle today's nav check. You make a mental note to get it looked at next time you're in port."

File: docs/developer-guide.md [hunk 2]
```diff
@@ -441,6 +441,18 @@ Modules follow the same LIST + function lookup pattern as cargo, locations, and
...
+
+- **Cargo Management System** (500€) — auto-files one paperwork chunk per day. Full (75%+): every day. Reduced (50-74%): odd-numbered trip days only. The paperwork task disappears from P3 once `PaperworkDone >= PaperworkTotal`.
```
Comment (spaceninja) [inline docs/developer-guide.md]: Similar to the auto-nav module, let's give the user some feedback that the cargo management system is degraded. How about whenever the cargo management system completes a paperwork chunk in a degraded state, we note that it's taking longer to complete its work: "The Cargo Management System pings, indicating it completed a chunk of paperwork for you, but you can't help but notice that it seems to be taking longer than normal. When you're in port, you think you should have a tech look at it."
